packages/abi-utils/lib/arbitrary.ts

```diff
@@ -0,0 +1,315 @@
+import * as fc from "fast-check";
+import faker from "faker";
+import { camelCase, pascalCase } from "change-case";
+
+import * as Types from "./types";
+
+export const Parameter = () =>
+  fc
+    .tuple(
+      fc.record({
+        name: ParameterName()
+      }),
+      TypeRecord()
+    )
+    .map(records => records.reduce((a, b) => ({ ...a, ...b }), {}));
```

Comment:
Kind of overkill for combining two things rather than an indefinite number, don't you think?